starter-code/server.js

```diff
@@ -172,7 +201,9 @@ function loadArticles() {
 
 function loadDB() {
   // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
-  // Put your response here...
+
+  // Put your response here... Queries are being made so it corresponds with number 3. CREATE is the part of CRUD that is being utilized. I could see Article.insertRecord being the method in article.js that is interacting with this function, but again I'm not entirely certain.
+
```

Comment:
loadDB is called when the server.js is executed. I think your original answer said loadArticles was calling loadDB, but loadArticles is actually called in the then block in loadDB.